test/CMakeLists.txt

```diff
@@ -7,7 +7,8 @@ function(add_tests)
             "${name}.cpp"
             LIBRARIES
             warnings
-            stdx)
+            stdx
+            rapidcheck_catch)
```

Comment:
**issue**: Linking with rapidcheck_catch should not be necessary - the cmake machinery already does this at https://github.com/intel/cicd-repo-infrastructure/blob/main/cmake/test.cmake#L110

Reply:
ok 👍🏽